recipes/libbasisu/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class LibBasisUniversalConan(ConanFile):
+    name = "libbasisu"
+    description = "Basis Universal Supercompressed GPU Texture Codec"
+    homepage = "https://github.com/BinomialLLC/basis_universal"
+    topics = ("conan", "basis", "textures", "compression")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "sse": [True, False],
+        "x64": [True, False],
+        "zstd": [True, False],
+        "no_iterator_debug_level": [True, False]
```

Comment:
I suppose this option sets `_ITERATOR_DEBUG_LEVEL`.
This means that this option is only relevant for `Visual Studio`.
So please remove this option in `config_options` when `self.settings.compiler != "Visual Studio"`.